.github/labeler.yml

```diff
@@ -0,0 +1,18 @@
+core:
+  - quantum/**/*
+  - tmk_core/**/*
+  - drivers/**/*
+  - tests/**/*
+  - util/**/*
+keymap:
+  - users/**/*
+  - layouts/**/*
+  - keyboards/**/keymap/**/*
```

Comment:
```suggestion
  - keyboards/**/keymaps/**/*
```

Reply:
Is it possible to exclude `default` and `via` here?

Reply:
Maybe in the future but not right now as it has the same limitations as the labeler actions